libraries/Microsoft.Bot.Connector/Microsoft.Bot.Connector.csproj

```diff
@@ -27,8 +27,11 @@
 
   <ItemGroup>
     <Compile Remove="node_modules\**" />
+    <Compile Remove="Skills\**" />
```

Comment:
Is this needed? or checked in by mistake? (I don't really get the node_modules either but it was there, we may be able to remove all the things in this ItemGroup?)